src/zrc/api/tests/test_auth.py

```diff
@@ -288,86 +292,31 @@ def test_list_zaakinformatieobject_limited_to_authorized_zaken(self):
-            vertrouwelijkheidaanduiding=zio1.zaak.vertrouwelijkheidaanduiding
-        ):
-            url = reverse('zaakinformatieobject-list', kwargs={'zaak_uuid': zio1.zaak.uuid})
-            zio1_url = reverse(zio1, kwargs={'zaak_uuid': zio1.zaak.uuid})
-
-            response = self.client.get(url)
+        url = reverse('zaakinformatieobject-list')
 
-            self.assertEqual(response.status_code, status.HTTP_200_OK)
-            response_data = response.json()
-            self.assertEqual(len(response_data), 1)
-            self.assertEqual(
-                response_data[0]['url'],
-                f"http://testserver{zio1_url}"
-            )
-
-        # not allowed to see these
-        for zio in (zio2, zio3):
-            with self.subTest(
-                zaaktype=zio.zaak.zaaktype,
-                vertrouwelijkheidaanduiding=zio.zaak.vertrouwelijkheidaanduiding
-            ):
-                url = reverse('zaakinformatieobject-list', kwargs={'zaak_uuid': zio.zaak.uuid})
-
-                response = self.client.get(url)
-
-                self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
-
-    def test_detail_zaakinformatieobject_limited_to_authorized_zaken(self):
-        # must show up
-        zio1 = ZaakInformatieObjectFactory.create(
-            zaak__zaaktype='https://zaaktype.nl/ok',
-            zaak__vertrouwelijkheidaanduiding=VertrouwelijkheidsAanduiding.openbaar
-        )
-        # must not show up
-        zio2 = ZaakInformatieObjectFactory.create(
-            zaak__zaaktype='https://zaaktype.nl/not_ok',
-            zaak__vertrouwelijkheidaanduiding=VertrouwelijkheidsAanduiding.openbaar
-        )
-        # must not show up
-        zio3 = ZaakInformatieObjectFactory.create(
-            zaak__zaaktype='https://zaaktype.nl/ok',
-            zaak__vertrouwelijkheidaanduiding=VertrouwelijkheidsAanduiding.vertrouwelijk
-        )
-
-        with self.subTest(
-            zaaktype=zio1.zaak.zaaktype,
-            vertrouwelijkheidaanduiding=zio1.zaak.vertrouwelijkheidaanduiding
-        ):
-            url = reverse(zio1, kwargs={'zaak_uuid': zio1.zaak.uuid})
-
-            response = self.client.get(url)
-
-            self.assertEqual(response.status_code, status.HTTP_200_OK)
-
-        # not allowed to see these
-        for zio in (zio2, zio3):
-            with self.subTest(
-                zaaktype=zio.zaak.zaaktype,
-                vertrouwelijkheidaanduiding=zio.zaak.vertrouwelijkheidaanduiding
-            ):
-                url = reverse(zio, kwargs={'zaak_uuid': zio.zaak.uuid})
+        response = self.client.get(url)
 
-                response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data), 1)
 
-                self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+        zaak_url = reverse('zaak-detail', kwargs={'uuid': zio1.zaak.uuid})
```

Comment:
you could use `reverse(zio1.zaak)` here, which will figure out the detail url as well (since it's a model instance)